conventions/code-reviews.md

```diff
@@ -0,0 +1,8 @@
+# Code Reviews
+
+1. The implementer is responsible to drive the feature, and so to ask other developers for reviews
```

Comment:
> The implementer is responsible for driving the feature through the project board and if needed, the implementor should ask peers for code review on slack.

Reply:
`implementer` or `implementor`?
Also, I would not say slack. I would leave the channel of communication ambiguous so people can decide which one works better for them.

Reply:
Apparently [either is okay](https://english.stackexchange.com/questions/195699/implementor-vs-implementer)